website/docs/r/media_convert_queue.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: "aws"
```

Comment:
Since this pull request was introduced, we now require a subcategory field in the frontmatter which drives the sidebar navigation in the [Terraform Registry documentation](https://registry.terraform.io/providers/hashicorp/aws):

```suggestion
subcategory: "MediaConvert"
layout: "aws"
```